### PR TITLE
Add basic getting started info

### LINKED ISSUE #2: How does one run this application?

### PR DESCRIPTION
**Pull request by SteveMarshall**
Not everyone can intuit how things should be used from the presence of
a Vagrantfile, and this is for everyone. This fixes #2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
+
+Requirements:
+
+- [Vagrant](https://www.vagrantup.com/) v1.5+
+- [VirtualBox](https://www.virtualbox.org) v4+ (or VMWare Fusion and
+  the Vagrant plugin)
+
+1. Run `vagrant up`. At the end of the output, there will be a notice
+   detailing the application port and docker port (assuming you have
+   docker installed)
```
Comment (matharden) [inline README.md]: Maybe a capital 'D' for Docker?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
 Money to Prisoners public application
+
+## Getting started
+
+Requirements:
+
+- [Vagrant](https://www.vagrantup.com/) v1.5+
+- [VirtualBox](https://www.virtualbox.org) v4+ (or VMWare Fusion and
+  the Vagrant plugin)
+
```
Comment (matharden) [inline README.md:14]: Possibly another heading here, as the following list are not part of the application requirements.

**Comment by SteveMarshall** [comment]
@matharden Added some tweaks.

**Comment by matharden** [comment]
:+1: :shipit:

**Pull request merged**